docs/standard/base-types/type-conversion.md

```diff
@@ -5,7 +5,7 @@ ms.date: "03/30/2017"
 ms.prod: ".net"
 ms.reviewer: ""
 ms.suite: ""
-ms.technology:v
+ms.technology: v
```

Comment:
I don't think the technology should be "v".

Though since this Pull Request clearly improves the article, I think it's okay to merge it and fix that later.